### PR TITLE
Fix dissymmetry in the appearance of "I'm feeling adventurous"

### LINKED ISSUE #209: Dissymmetry in the appearance of the website.
The elements containing the text **"Bugs & Issues"** and **"I'm feeling adventurous"** are aligned in the same horizontal line on the website. 

While, there's no padding around **"Bugs & Issues"**, we can see that padding has been provided around the element **"I'm feeling adventurous."** This is causing a dissymmetry in the appearance of the website.

The following screenshots explain the above : 

1.  Dissymmetry :

 full

2.  No padding on **Bugs & Issues**:

    bugs
 

3.  Padding on **I'm feeling adventurous**:

 padding

If suitable, then I would like to work on this issue.

### PR DESCRIPTION
**Pull request by sonakshisaxena1**
Removed the right padding of **I'm feeling adventurous** element, henceforth, enhancing the website's look by adding symmetry. 

Padding around the element - 

solved

Final appearance -

screen shot 2018-08-28 at 2 20 20 am

### CODE REVIEW

File: src/components/TasksTable/index.jsx [hunk 1]
```diff
@@ -100,6 +100,9 @@ const assignments = Object.values(ASSIGNEE);
     flex: 1,
     whiteSpace: 'nowrap',
   },
+  adventurousButton: {
+    paddingRight: 0,
```
Comment (helfi92) [inline src/components/TasksTable/index.jsx]: There seems to be an issue when hovering on the button (no white space on the right):

screen shot 2018-08-27 at 5 33 36 pm

Comment (sonakshisaxena1) [reply src/components/TasksTable/index.jsx]: Oh, I missed that!
I’ll have a look at this.

Comment (helfi92) [reply src/components/TasksTable/index.jsx]: No worries :)

Comment (sonakshisaxena1) [reply src/components/TasksTable/index.jsx]: @helfi92 
I think I have fixed this. 
Please review it again :)

Before hovering:

screen shot 2018-08-29 at 11 44 29 am

On hovering: 

screen shot 2018-08-29 at 11 43 53 am

**Comment by sonakshisaxena1** [comment]
I had updated the branch, but it didn't work. 
I will try to rebase it before you review, again.
Sorry for the inconvenience.

**Comment by sonakshisaxena1** [comment]
@helfi92 I am afraid that I am unable to find any margins around the button. 
I wonder if setting a margin would help us solve this issue, since it will further increase the distance of the text from the right. 
Can you guide me on this?

**Comment by helfi92** [comment]
@sonakshisaxena1 Now that I think about it, given that the hover effect takes more space, the current layout is symmetric. One way to make it more clear in the UI is to change the variant of the button.

```jsx
<Button
  variant="outlined"
  ...
>
  I’m Feeling Adventurous
</Button>
```

This should help make it look nicer :)

**Comment by sonakshisaxena1** [comment]
Definitely! :D
I'll make a new commit regarding this. Thankyou!

**Pull request merged**